### PR TITLE
Allow setting projectid for customer gateway

### PR DESCRIPTION
**Pull request by schans**
Hi all!

Please find my humble (trivial) pull request for optionally setting the projectid for the customer vpn gateway.

Let me know what I can do to help to get this in!

Thanks
Bart

### CODE REVIEW

File: website/docs/r/vpn_customer_gateway.html.markdown [hunk 1]
```diff
@@ -49,6 +49,9 @@ The following arguments are supported:
 * `ike_lifetime` - (Optional) The IKE lifetime of phase 2 VPN connection to this
     VPN Customer Gateway in seconds (defaults 86400)
 
+* `project` - (Optional) The name or ID of the project to deploy this
+    instance to. Changing this forces a new resource to be created.
```
Comment (svanharmelen) [inline website/docs/r/vpn_customer_gateway.html.markdown]: I noticed a few resources have this exact description while others have a more specific one. So think that something like `The name or ID of the project to create this VPN Customer Gateway in. Changing this forces a new resource to be created.` would be just a bit niced :wink"

**Comment by schans** [comment]
Hi @svanharmelen !

Thanks again for reviewing! I'll update the patch with your suggestions.

Thanks
Bart

**Comment by schans** [comment]
Hi @svanharmelen! I have updated the pull request with your suggestions. Please let me know if I can do anything else to help out :-)

**Comment by svanharmelen** [comment]
LGTM @schans! Thanks!

**Comment by schans** [comment]
Cool!

**Pull request merged**